patches/chrome-android-BUILD.gn.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
-index 02f17d2b4e93d27964fe6663c4feddccc9634619..bd0459db766702d7bc0127914e687421e34852da 100644
+index 02f17d2b4e93d27964fe6663c4feddccc9634619..6580087f7bdd344d7e4d00464a9224e99cc888f7 100644
 --- a/chrome/android/BUILD.gn
 +++ b/chrome/android/BUILD.gn
-@@ -38,7 +38,7 @@ if (enable_vr) {
+@@ -2,6 +2,7 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/android/brave_java_sources.gni")
```

Comment:
remember we want these to be extensible so I would probably just make this `//brave/build/android/config.gni`

Reply:
and maybe it should be imported in `//build/config/android/config.gni` so it gets included in more places?

Reply:
That is an import to include custom java files in a way that chromium does it a separate file for java files.

Reply:
done